hal/lib/athena/AnalogGyro.cpp

```diff
@@ -20,15 +20,15 @@ struct AnalogGyro {
   HAL_AnalogInputHandle handle;
   float voltsPerDegreePerSecond;
   float offset;
-  uint32_t center;
+  signed int center;
```

Comment:
The reason for this, is I pulled the type directly from the FPGA function that uses it, and that is what it directly specifies. I can make int32_t, but it seemed like a decent idea to make it match the FPGA functions. I did it in a few other places as well.